src/prefect/tasks/redis/redis_tasks.py

```diff
@@ -0,0 +1,214 @@
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+import redis
+
+
+class RedisSet(Task):
+    """
+    Task for setting a Redis key-value pair.
+
+    Args:
+        - connection (Redis Connection , optional): if not provided, connection will be established using
+            host, port, db, and password parameters
+        - host (str, optional): name of Redis host, defaults to 'localhost'
+        - port (int, optional): Redis port, defaults to 6379
+        - db (int, optional): redis database index, defaults to 0
+        - password (str, optional): Redis password, defaults to None
+        - redis_key (optional): Redis key to be set, can be provided at initialization or runtime
+        - redis_val (optional): Redis val to be set, can be provided at initialization or runtime
```

Comment:
what are the types of `redis_key` and `redis_val`?

Reply:
Did my best to clarify this, let me know if you think it can be improved

Reply:
Looks good to me!